linode/helper/logging.go

```diff
@@ -0,0 +1,53 @@
+package helper
+
+import (
+	"context"
+	"net/http"
+	"os"
+
+	"github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+)
+
+const (
+	APILoggerSubsystem = "linodego-requests"
+	EnvAPILogger       = "TF_LOG_PROVIDER_LINODE_REQUESTS"
```

Comment:
The environmental variable in the doc above is `TF_LOG_PROVIDER_LINODE`. Should we change it here?

Reply:
Oops, looks like a typo. I'll fix this tomorrow :+1: